### ISSUE TITLE
Expore: Graph severity issue

### ISSUE THREAD

**Issue by ventris** [grafana/grafana#14481]
### What Grafana version are you using?
grafana:master docker image
### What datasource are you using?
Loki
### What OS are you running grafana on?
Debian
### What did you do?
When using the Explore UI, there is an graph that shows data over time. And there are some icons below it that represents the severity.

![](https://i.imgur.com/S9kwCkJ.png)

This severity is only picked from keywords which is troublesome in those cases where one is exposed to events that include the keywords in question without having that severity on the event.

In the image listed above I tried to do dns lookups for error.com and the Explore UI tags this with an error tag and colors the event red even though my receive code from the DNS is NOERROR it still gets miss-tagged as error.

This is a great feature in general but as in my testcase this creates some issues as the event had no errors but grafana tags it as such anyway. Maybe there should be some requirement for how the severity is parsed? Or an ability to disable this labeling somehow.

I found this when dropbox is periodically looking up dl-debug.dropbox.com and tags the event as debug in Explore UI.

**Comment by davkal** [comment]
This may be done in the backend soon. Until then, I'm hesitant about changing the JS parser logic.

**Comment by ventris** [comment]
Sounds like a good plan to change it in the backend.

**Comment by aocenas** [comment]
@ifrost can we close this with the full range histogram?

**Comment by ifrost** [comment]
@aocenas yes, full range histogram won't get the severity value from the message but will use `level` label instead (which could also be extracted with `logfmt` operator).
